lms/static/js/edxnotes/views/tabs/course_structure.js

```diff
@@ -4,10 +4,12 @@ define([
     'gettext', 'underscore', 'js/edxnotes/views/note_group', 'js/edxnotes/views/tab_panel',
     'js/edxnotes/views/tab_view'
 ], function (gettext, _, NoteGroupView, TabPanelView, TabView) {
+    var view = "Location in Course";
     var CourseStructureView = TabView.extend({
+        view: 'Location in Course',
```

Comment:
@muzaffaryousaf Oh, right!! Yes, I will correct that. Sorry, I was misreading the diff.